### DISCUSSION TITLE
skip some frames(vid_stride) implementation

### DISCUSSION THREAD

**Discussion by aerenkaradag** [roboflow/supervision#35] [Q&A]
How would that be possible, if I want to skip some frames(vid_stride) in the video because of slow video processing? Like, look once in every 4 frames.

I'm talking about that YT video: https://youtu.be/l_kf9CfZ_8M

**Comment by SkalskiP** [comment]
@aerenkaradag We could add the argument - `interval: int` to the `get_video_frames_generator` function so that it would return every nth frame.

```python
def get_video_frames_generator(source_path: str, interval: int = 1) -> Generator[np.ndarray, None, None]:
    ...
```
It would be equal to `1` by default so that you'd read all frames, but when you'd set it to `2`, you'd get every 2nd frame and so on.

Would that solve your issue?
